src/infrasys/system.py

```diff
@@ -33,16 +33,20 @@
 )
 from infrasys.time_series_manager import TimeSeriesManager, TIME_SERIES_KWARGS
 from infrasys.time_series_models import SingleTimeSeries, TimeSeriesData, TimeSeriesMetadata
+from infrasys.utils.sqlite import backup, create_in_memory_db, restore
 
 
 class System:
     """Implements behavior for systems"""
 
+    DB_FILENAME = "time_series_metadata.db"
```

Comment:
We should probably change the name of this. At serialization time, we store this file in a time series directory. So, it could be mildly confusing when supplemental attributes are added. We would have to add support for deserializing old systems. I'm voting that we delay this for now.